app.js

```diff
@@ -1,93 +1,109 @@
 'use strict';
-var middleName = prompt('Can you guess if my middle name is Charles? \(Yes or No\)');
+
+function middleName() {
+  var middleName = prompt('Can you guess if my middle name is Charles? \(Yes or No\)');
   middleName.toLowerCase();
-if (middleName === 'no') {
+  if (middleName === 'no') {
     alert('Wow, you guessed that my middle name is not Charles. It\'s actually Clifford.');
     console.log(middleName);
-} else {
+  } else {
     alert('You are right, my middle name is actually Clifford.');
     console.log(middleName);
-}
+  }
+} middleName();
```

Comment:
your function declarations should end with a semicolon, and your function call should be on its own line